tests/test_tensor.py

```diff
@@ -370,6 +370,61 @@ def test_boolean_mask(backend):
     )
 
 
+@pytest.mark.skip_jax
+def test_percentile(backend):
+    tb = pyhf.tensorlib
+    a = tb.astensor([[10, 7, 4], [3, 2, 1]])
+    assert tb.tolist(tb.percentile(a, 0)) == 1
+
+    assert tb.tolist(tb.percentile(a, 50)) == 3.5
+    assert tb.tolist(tb.percentile(a, 100)) == 10
+    assert tb.tolist(tb.percentile(a, 50, axis=1)) == [7.0, 2.0]
+
+
+# FIXME: PyTorch doesn't yet support interpolation schemes other than "linear"
+# c.f. https://github.com/pytorch/pytorch/pull/59397
+@pytest.mark.skip_pytorch
+@pytest.mark.skip_pytorch64
+@pytest.mark.skip_jax
```

Comment:
Skipping PyTorch as interpolation kwargs aren't implemented yet as of `torch` `v1.10.0`. Skipping JAX so as to have separate test below given floating point issues with `"linear"` option.